app/Http/Middleware/User.php

```
@@ -0,0 +1,31 @@
+use Symfony\Component\HttpFoundation\Response;
+
+class User
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        $user = Auth::user();
```

PHPMD (2.15.0)

Avoid using static access to class '\Auth' in method 'handle'. (Clean Code Rules)

(StaticAccess)